src/app/services/map.service.ts

```diff
@@ -180,11 +180,11 @@ export class MapService {
       ).subscribe(params => {
         this.map.getView().fit(params.bounds, { nearest: true });
       });
-    });
-
-    this.orderStatus.subscribe((status) => {
-      this.validationStatus = status;
-      this.updateAreaTooltip();
+    }).then(() => {
+      this.orderStatus.subscribe((status) => {
+        this.validationStatus = status;
+        this.updateAreaTooltip();
```

Comment:
Maybe add another user detection here, once everything is setup ?

To cover the case where the state isLoggedIn$ is not updated correcly